tests/headers/opaque-template-inst-member.hpp

```diff
@@ -3,10 +3,18 @@
 template<typename T>
 class OpaqueTemplate {
     T mData;
-    bool mCannotDebug[40];
+    bool mCannotDebug[400];
 };
 
+// This should not end up deriving Debug because its `mBlah` field cannot derive
+// Debug because the instantiation's definition cannot derive Debug.
 class ContainsOpaqueTemplate {
```

Comment:
Part of the point of the test is that we derived debug for the instantiation if we could, could you add that back, maybe as another different test-case?